.env.j2

```diff
@@ -5,13 +5,6 @@ FLASK_ENV=production
 # Change the following values for local development.
 SECRET_KEY={{ secret_key }}
 
-# Change the following values depending on your Trello environment.
-TRELLO_API_KEY={{ trello_api_key }}
-TRELLO_TOKEN={{ trello_token }}
-TRELLO_BASE_URL=https://api.trello.com/1/
-
-# Change the following values depending on the Trello board being modified.
-BOARD_ID={{ board_id }}
-NOT_STARTED_LIST_ID={{ not_started_list_id }}
-IN_PROGRESS_LIST_ID={{ in_progress_list_id }}
-DONE_LIST_ID={{ done_list_id }}
+# Database connection
+DB_NAME={{ db_name }}	
+DB_PRIMARY_CONNECTION_STRING={{ db_primary_connection_string }}

```

Comment:
Nice work updating this, but don't feel you need to for future exercises; it is a good habit but we won't be using Ansible in future